sklearn/preprocessing/_encoders.py

```diff
@@ -739,7 +741,7 @@ def __init__(
         drop=None,
         sparse="deprecated",
         sparse_output=True,
-        dtype=np.float64,
+        dtype=np.uint8,
```

Comment:
Can you revert all the changes in `_encoders.py`? The change is not related to the title of this PR. (memoryviews + cnp.ndarray)